onnx/defs/training/defs.cc

```diff
@@ -308,35 +278,28 @@ ONNX_TRAINING_OPERATOR_SET_SCHEMA(
         .Input(
             0,
             "Inputs",
-            "The values fed into the graph specified by the \"graph_name\" attribute. "
-            "The i-th input is bound to the input named \"input_names[i]\" in the "
-            "called graph.",
+            "Inputs fet to the invoked graph. "
+            "The i-th input here goes to the i-th input of the "
+            "invoked graph. To omit an optional input in this field, "
+            "the user can omit it or use an empty string.",
             "T",
             OpSchema::Variadic,
             false)
         .Output(
             0,
             "Outputs",
-            "The outputs produced by the called graph. Its i-th value is bound to the "
-            "output named \"output_names[i]\" in the inference graph.",
+            "The outputs generated by the called graph. "
+            "Its i-th value is bound to the i-th output of the called graph. "
+            "Similar to the inputs, all outputs are optional.",
             "T",
             OpSchema::Variadic,
             false)
         .Attr(
             "graph_name",
             "The invoked graph's name. "
-            "Currently, the only allowed value is \"ModelProto.graph.name\".",
+            "The only allowed value is \"ModelProto.graph.name\" "
+            "in the ONNX model format.",
```

Comment:
I'm not quite sure how you can disambiguate, but this could be read as `graph_name = "ModelProto.graph.name"`, rather than the value of `ModelProto.graph.name`

Maybe
> The only allowed value is the name of the inference graph, which is stored in \"ModelProto.graph.name\" in the ONNX model format.

Reply:
Just a minor nit: I think the documentation of the operator is not the right place for capturing such constraints. Ideally, the set of "callable graphs" should be defined/constrained in TrainingInfoProto; may be one day we might even generalize it to allow more graphs to be called.

Reply:
@TMVector, your words are directly copied to the spec. Thank you.

@gramalingam, At the end of `TrainingInforProto`'s documentation, I explicitly indicates the uses of GraphCall can only happen in TrainingInfoProto.algorithm. Also, I add one sentence to ModelProto.graph --- `Operators in ai.onnx.training` domain cannot be included in this field.